### PR TITLE
copy-edit-problem-solving-technique

### LINKED ISSUE #455: https://github.com/Umuzi-org/ACN-syllabus/tree/develop/content/language-agnostic/problem-solving-technique/_index.md

### PR DESCRIPTION
**Pull request by asradebe**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [ ] I ran the hugo server and looked at my changed in the browser with my own eyes
- [ ] I ran the linter and there were no errors
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: content/language-agnostic/problem-solving-technique/_index.md [hunk 1]
```diff
@@ -11,57 +11,57 @@ As a coder, your job is to write code that solves problems. Problems that nobody
...
-Try to actually do the problem by hand. Or perhaps a smaller version of the problem.
+Try to do the problem by hand. Or perhaps a smaller version of the problem.
 
-For example, if you are instructed to do something interesting to an array with length 1000, see what you can manually do to an array of length 5 that is equivalent. Try solve it a few times with different inputs and see if you can find a pattern that you can break down :)
+For example, if you are instructed to do something interesting to an array with length 1000, see what you can manually do to an array of length 5 that is equivalent. Try to solve it a few times with different inputs and see if you can find a pattern that you can break down :)
 
 ## Beware of the edge cases
 
-Edge case is kinda a funny thing to understand. Basically you need to think of extreme values and see if your code can handle that.
+An edge case is kinda a funny thing to understand. You need to think of extreme values and see if your code can handle that.
```
Comment (JacobUmuzi) [inline content/language-agnostic/problem-solving-technique/_index.md]:
```suggestion
An edge case is kind of a funny thing to understand. You need to think of extreme values and see if your code can handle that.
```

File: content/language-agnostic/problem-solving-technique/_index.md [hunk 2]
```diff
@@ -11,57 +11,57 @@ As a coder, your job is to write code that solves problems. Problems that nobody
...
 
 For example:
 
 - If you are writing code for a game, how do you deal with things if the players tie?
 - If you need to write code that accepts an integer, what if the value is zero?
-- What if you need to do some processing on an input string and the string is empty? What if it's really really long?
+- What if you need to do some processing on an input string and the string is empty? What if it's long?
```
Comment (JacobUmuzi) [inline content/language-agnostic/problem-solving-technique/_index.md:34]:
```suggestion
- What if you need to do some processing on an input string and the string is empty? What if it's very long?
```

File: content/language-agnostic/problem-solving-technique/_index.md [hunk 3]
```diff
@@ -11,57 +11,57 @@ As a coder, your job is to write code that solves problems. Problems that nobody
...
 
-The resources above are great resources for coding related problem solving techniques. But, problem solving is not unique only to coding. It is a skill to be practiced in everyday life. Please checkout the following content from MIT:
+The resources above are great resources for coding related problem-solving techniques. But, problem-solving is not unique only to coding. It is a skill to be practised in everyday life. Please check out the following content from MIT:
 
 - https://ccmit.mit.edu/problem-solving/
 
 ## Practice
 
-Of course, reading about code and watching other people code is great. But if you personally want to get good at this stuff: Practice practice practice.
+Of course, reading about code and watching other people code is great. But if you want to get good at this stuff: Practice practice practice.
```
Comment (JacobUmuzi) [inline content/language-agnostic/problem-solving-technique/_index.md]:
```suggestion
Of course, reading about code and watching other people code is great. But if you want to get good at this stuff: Practise practise practise.
```

**Review by JacobUmuzi — CHANGES_REQUESTED** [review]

File: content/language-agnostic/problem-solving-technique/_index.md (hunk 3 above)
Comment (asradebe) [reply content/language-agnostic/problem-solving-technique/_index.md]: Grammarly throws an error, lol lol

File: content/language-agnostic/problem-solving-technique/_index.md (hunk 2 above)
Comment (asradebe) [reply content/language-agnostic/problem-solving-technique/_index.md:34]: grammarly throws an error here too

File: content/language-agnostic/problem-solving-technique/_index.md [hunk 4]
```diff
@@ -11,57 +11,57 @@ As a coder, your job is to write code that solves problems. Problems that nobody
...
```
Comment (JacobUmuzi) [inline content/language-agnostic/problem-solving-technique/_index.md]:
```suggestion
Of course, reading about code and watching other people code is great. But if you want to get good at this stuff: Practise practise practise.
```

**Review by JacobUmuzi — APPROVED** [review]

**Pull request merged**